### PR TITLE
Get current locale

### PR DESCRIPTION
**Pull request by ichan-mb**
add a property to get current locale information

This PR contains:
- [ ] Changelog
- [ ] Documentation
- [ ] Tests

### CODE REVIEW

File: Source/FuseJS/Environment.uno [hunk 1]
```diff
@@ -64,13 +66,70 @@ namespace FuseJS
...
+						NSRange localeRange = NSMakeRange(underscoreIndex.location, atSignIndex.location - underscoreIndex.location);
+						language = [NSString stringWithFormat: @"%@%@", language, [localeId substringWithRange: localeRange]];
+					}
+				}
+			}
+
+			return [language stringByReplacingOccurrencesOfString: @"_" withString: @"-"];
+		@}
+
+		static extern(!Mobile) string GetCurrentLocale() {
```
Comment (mortend) [inline Source/FuseJS/Environment.uno]: Style: Braces should go on their own line when defining methods.

File: Source/FuseJS/Environment.uno [hunk 2]
```diff
@@ -64,13 +66,70 @@ namespace FuseJS
...
+			return "en-EN";
```
Comment (mortend) [inline Source/FuseJS/Environment.uno]: Are you sure `en-EN` is a valid locale? `en-US` sounds more familiar to me.

File: Source/FuseJS/Environment.uno [hunk 3]
```diff
@@ -64,13 +66,70 @@ namespace FuseJS
 			AddMember(new NativeProperty<bool, bool>("dotnet", defined(DOTNET))); // Undocumented for testing use only
 			AddMember(new NativeProperty<bool, bool>("host_mac", defined(HOST_MAC))); // Undocumented for testing use only
 			AddMember(new NativeProperty<string, string>("mobileOSVersion", GetMobileOSVersion));
+			AddMember(new NativeProperty< string, object >("locale", GetCurrentLocale));
```
Comment (mortend) [inline Source/FuseJS/Environment.uno]: Style: Please drop spaces inside `<` and `>`.

File: Source/FuseJS/Environment.uno (hunk 1 above)
Comment (ichan-mb) [reply Source/FuseJS/Environment.uno]: oops! fix on the way

File: Source/FuseJS/Environment.uno (hunk 3 above)
Comment (ichan-mb) [reply Source/FuseJS/Environment.uno]: oops! fix on the way

File: Source/FuseJS/Environment.uno (hunk 2 above)
Comment (ichan-mb) [reply Source/FuseJS/Environment.uno]: Fixed

**Pull request merged**